parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -293,6 +293,27 @@ configureCNIIPTables() {
     fi
 }
 
+configureKubeletSecureTLSBootstrap() {
+    # default AAD resource here so we can minimze bootstrap contract surface
+    AAD_RESOURCE="6dae42f8-4368-4678-94ff-3960e28e3630"
```

Comment:
If user doesn't set CUSTOM_SECURE_TLS_BOOTSTRAP_AAD_RESOURCE, AAD_RESOURCE is defaulted to 6dae42f8-4368-4678-94ff-3960e28e3630. Does it mean this AAD_RESOURCE is always available in every user's subscription?

Reply:
yes, as far as I know - this is the app ID of AKS's AAD server, I've spoken with Baichao and Weinong about this:

https://azure.github.io/kubelogin/concepts/aks.html

Reply:
though will be a good idea to confirm this is correct gov environments as well